### PR TITLE
[pytest.yml] fix ci

### PR DESCRIPTION
**Pull request by diivm**
should be a hotfix

### CODE REVIEW

**Comment by diivm** [comment]
Mirror sync in progress. Maybe try later

**Comment by kunaltyagi** [comment]
Requires review?

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -27,53 +27,56 @@ jobs:
...
+      - name: Add llvm keys
+        run: |
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          echo 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
+          echo 'deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
+      - name: Install libclang and its python bindings
+        run: |
+          sudo apt-get update
+          sudo apt-get cache
+          sudo apt-get install -y libclang-11-dev python3-clang-11
```
Comment (aPonza) [inline .github/workflows/pytest.yml]: This being the only diff if ignoring whitespace changes, I'm prone to ask:
- why remove `-y` from update?
- what is `apt-get cache`? On my machine:
```
$ apt-get cache
E: Invalid operation cache
```

Comment (diivm) [reply .github/workflows/pytest.yml]: @kunaltyagi @aponza Please ignore this PR, it's just for re-running and trying out some stuff for fixing broken CI.

> why remove -y from update?

It doesn't require it.

> what is apt-get cache

A mistake :joy:

**Comment by diivm** [comment]
> Requires review?

Nope, fixed.

**Pull request merged**